.changeset/mighty-feet-approve.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": patch
+---
+
+Card (default) design changed to include border
```

Comment:
Can you include why it was changed please.

Reply:
is this ok?